src/main/java/com/onaple/itemizer/utils/ItemBuilder.java

```diff
@@ -80,7 +92,7 @@ private static BlockState addTraits(BlockType blockType, Map<String, String> tra
         setCustomDatamanipulators(itemBean);
         setNbt(itemBean);
         applyCustomLoreAppender();
```

Comment:
ItemBean.getNbt has to return a collection which may use comparators, use TreeSet